developer/building/qubes-builder.md

```diff
@@ -71,6 +79,10 @@ Additionally, if building with signing enabled (NO\_SIGN is not set), you must a
 It is also recommended that you use an empty passphrase for the private key used for signing.
 Contrary to a popular belief, this doesn't affect your key or sources security -- if somebody compromised your system, then the game is over anyway, whether you have used an additional passphrase for the key or not.
 
+You will also need include your key in each chroot used for the build
+(which for this reason have to be generated first), by running `rpm
+--import` in each of those chroots.
```

Comment:
I think this is clumsy.
You will need to generate each chroot used for the build, and include your key in each, by running `rpm --import`

This has the advantage of matching the instructions you provide. Again, specify that this applies only to Fedora based chroots, or provide instructions for other templates.

Reply:
Looks like signing of debian packages will have to be added separately :)

Reply:
>  signing of debian packages 

You'll know that this is rarely done in Debian, and there is not a standard on how to do it.